index.html

```diff
@@ -289,6 +289,21 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <h2 class="contact_heading">Contact me</h2>
+        <div class="contact_form_section">
+          <p class="contact_text">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs
+            coding. I’d love to help with it
+          </p>
+          <form method="post">
+            <input class="form_input" type="text" name="fullname" id="fullname" maxlength="30" placeholder="Enter your name here" required>
+            <input class="form_input" name="email" id="email" placeholder="Enter your email here" required>
+            <textarea class="form_input" name="message" id="message" cols="30" rows="7" maxlength="500" placeholder="Write your message here" required></textarea>
+            <button class="submission" type="submit">Get In Touch</button>
+          </form>
```

Comment:
- [ ] It is part of the requirement to submit the form data upon submit button click to [formspree service](https://formspree.io/html). So, please include this functionality.